### PR TITLE
feat: add link to the github repository

### PR DESCRIPTION
**Pull request by sanjibansg**
This PR adds a link to the GitHub repository on the webpage. It will be helpful for new developers to locate the code repository for contribution.

This should resolve Issue #18 

Current visual:
Screenshot 2023-05-24 at 2 19 51 AM

### CODE REVIEW

File: client/src/App.vue [hunk 1]
```diff
@@ -8,20 +8,58 @@ import SubstraitGraph from "@/components/SubstraitGraph.vue";
...
+          <div class="col-4">
+            <br /><br />
+            <div
+              class="d-flex justify-content-end"
+              style="text-align: right; padding-bottom: 0%; margin-right: 1.5%"
+            >
+              <img
+                height="20"
+                width="20"
+                src="https://cdn.simpleicons.org/github/white"
```
Comment (sanjibansg) [inline client/src/App.vue:42]: Here, the GitHub logo hosted by [simpleicons.org](https://simpleicons.org/) is used. Will that be okay, or do we need a better option?

Comment (sanjibansg) [reply client/src/App.vue:42]: They have some free-to-use icons I believe with a legal disclaimer for use, might have to check their usage, or need to give attribution. 
https://github.com/simple-icons/simple-icons

File: client/src/App.vue [hunk 2]
```diff
@@ -8,20 +8,58 @@ import SubstraitGraph from "@/components/SubstraitGraph.vue";
...
+                style="margin-top: 2px; margin-right: 4px"
+              />
+              <a
+                href="https://github.com/voltrondata/substrait-fiddle/"
```
Comment (sanjibansg) [inline client/src/App.vue:46]: I wanted to have something that [FastAPI](https://fastapi.tiangolo.com/) uses in their documentation webpages, and even [Substrait](https://substrait.io/). I am not very sure if they are built from scratch, or if we can use an official GitHub badge.

**Review by westonpace — CHANGES_REQUESTED** [review]
Can we center the link vertically so it's on the same vertical position as the "Substrait Fiddle" header?

**Comment by sanjibansg** [comment]
> Can we center the link vertically so it's on the same vertical position as the "Substrait Fiddle" header?

updated this,
Screenshot from 2023-06-15 01-53-08

**Review by westonpace — APPROVED** [review]
Thanks for centering that

**Pull request merged**